internal/serve/httphandler/payments_handler.go

```diff
@@ -0,0 +1,33 @@
+package httphandler
+
+import (
+	"net/http"
+	"wallet-backend/internal/data"
+	"wallet-backend/internal/serve/httperror"
+
+	"github.com/stellar/go/support/http/httpdecode"
+)
+
+type PaymentsHandler struct {
+	*data.Models
```

Comment:
❓ does this mean will be put all the data models in the payments handler including ones that it doesn't use?

Reply:
Can we just include `data.PaymentMode`l here?

Reply:
We could, but a handler or a service is likely to need access to different data models, so this reduces significantly the hassle to implement and maintain them.

Since models only contain a pointer to the DB, I don't see a problem with that, as it isn't resource wasteful.

Reply:
To me, it's less about wasting the resource. It's more about making it clear what the PaymentsHandler is going to need. For example, if we include many models that are not used by the PaymentsHandler, it could be confusing to some readers.

Reply:
The dependencies do become harder to understand when everything has a link to all dependencies. And maybe be harder to write tests if we need to provide all models to payments tests that otherwise don't need to use them.

Reply:
@leighmcculloch writing tests shouldn't get harder I believe, because they can be left `nil` if not necessary for the test.
But I don't have a strong preference here, although I do lean towards the all-in-one approach because of its easier maintainability imo.

Reply:
Is it your preference to have it like so? [82a2c97](https://github.com/stellar/wallet-backend/pull/6/commits/82a2c974ae9b0d356c2be3182304a3b739b21bb5)

Reply:
I think 82a2c97 will be easier to understand which model a dependency uses, but I don't feel strongly and defer to you both.